bundles/ch.elexis.core.model/src/ch/elexis/core/model/Stock.java

```diff
@@ -83,6 +96,23 @@ public void setDescription(String value) {
 		getEntityMarkDirty().setDescription(value);
 	}
 
+	@Override
+	public void setLocation(String value) {
+		getEntityMarkDirty().setLocation(value);
+	}
+
+	@Override
+	public void setResponsible(IContact value) {
+		if (value != null) {
+			if (value instanceof AbstractIdModelAdapter) {
```

Comment:
in Java 17 kannst du das zusammenfassen mit `if(value instanceof AbstractIdModelAdapter aida)`: 
1) instanceof checkt automatisch ob != null, 
2) du kannst danach die bereits gecastete Variable `aida` verwenden